public/app/features/logs/components/panel/LogLineDetailsFields.tsx

```diff
@@ -339,7 +339,8 @@ export const LogLineDetailsField = ({
         }
         return (
           <div className={styles.row} key={`${link.title}-${i}`}>
-            <div className={disableActions ? styles.linkNoActions : styles.link}>
+            {!disableActions && <div />}
```

Comment:
Label links should be displayed as:

<img width="719" height="442" alt="Demo" src="https://github.com/user-attachments/assets/eeb53f38-754b-45cb-8fd6-7882fe7b1bb4" />

Derived links should be displayed as:

<img width="712" height="129" alt="Derived" src="https://github.com/user-attachments/assets/94520d21-190b-411f-947a-f659fbdf3639" />

Reply:
Are we forcing the button on a new line by rendering an empty div? Can we avoid extra dom elements and fix with css?